chainspec.go

```diff
@@ -28,10 +28,11 @@ type ChainSpec struct {
 	// Must be set.
 	Version string
 
-	// GasAdjustment and NoHostMount are pointers in ChainSpec
+	// GasAdjustment, NoHostMount and UsingNewGenesisCommand are pointers in ChainSpec
 	// so zero-overrides can be detected from omitted overrides.
-	GasAdjustment *float64
-	NoHostMount   *bool
+	GasAdjustment          *float64
+	NoHostMount            *bool
+	UsingNewGenesisCommand *bool
```

Comment:
Since `ChainSpec` embeds `ChainConfig`, I don't think this is necessary on the `ChainSpec` itself.

Reply:
The reason I added it was what I assumed was the same reason GasAdjusment and NoHosMount were there (which are also in ChainConfig), to be able to have them nullable and thus overridable. But I have to admit I didn't quite understand the config structure here so I just mimicked how it was done for the above mentioned values.

Reply:
I updated it (and added a test). It seems like it works just fine without being there :)